pkgs/games/anki/bin.nix

```diff
@@ -57,7 +57,7 @@ let
     targetPkgs = pkgs: (with pkgs; [ xorg.libxkbfile krb5 ]);
 
     runScript = writeShellScript "anki-wrapper.sh" ''
-      exec ${unpacked}/bin/anki
+      exec ${unpacked}/bin/anki ${ lib.strings.escapeShellArgs commandLineArgs }
```

Comment:
```suggestion
      exec ${unpacked}/bin/anki "$@"
```

Reply:
That does already work for some reason, though from what I understand, permanent additions via `override` nor `overrideAttrs` are possible.

The OP mentioned in a chat that the have seen some similar technique for some other FHSenv based packages and I encouraged them to do a PR.